astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -8,14 +8,141 @@
 
 import numpy as np
 
-from ... units import dimensionless_unscaled, UnitsError
-from ... import log
-from ...extern.six import string_types
-from ..nduncertainty import IncompatibleUncertaintiesException
+from ..nduncertainty import NDUncertainty
+from ...units import dimensionless_unscaled
+from ...config import ConfigAlias
+from ...extern import six
+
+WARN_UNSUPPORTED_CORRELATED = ConfigAlias(
+    '0.4', 'WARN_UNSUPPORTED_CORRELATED', 'warn_unsupported_correlated',
+    'astropy.nddata.nddata', 'astropy.nddata')
```

Comment:
This can be removed completely. The actual configuration item is defined in `nddata/__init__.py`